core/include/scipp/core/element/arithmetic.h

```diff
@@ -14,13 +14,20 @@ namespace scipp::core::element {
 
 constexpr auto add_inplace_types =
     arg_list<double, float, int64_t, int32_t, Eigen::Vector3d,
+             std::tuple<scipp::core::time_point, int64_t>,
+             std::tuple<scipp::core::time_point, int32_t>,
              std::tuple<double, float>, std::tuple<int64_t, int32_t>,
              std::tuple<int64_t, bool>>;
 
-constexpr auto plus_equals =
-    overloaded{add_inplace_types, [](auto &&a, const auto &b) { a += b; }};
-constexpr auto minus_equals =
-    overloaded{add_inplace_types, [](auto &&a, const auto &b) { a -= b; }};
+constexpr auto plus_equals = overloaded{
+    add_inplace_types,
+    [](auto &&a, const auto &b) { a += b; },
+};
+
+constexpr auto minus_equals = overloaded{
+    add_inplace_types,
+    [](auto &&a, const auto &b) { a -= b; },
+};
```

Comment:
I cannot see a change here? Broken clang-format?

Reply:
hmm that's how my clang-format formatted these, but I'll revert.